src/cmd/services/m3dbnode/config/limits.go

```diff
@@ -27,7 +27,12 @@ type LimitsConfiguration struct {
 	// MaxRecentlyQueriedSeriesBlocks sets the upper limit on time series blocks
 	// count within a given lookback period. Queries which are issued while this
 	// max is surpassed encounter an error.
-	MaxRecentlyQueriedSeriesBlocks *MaxRecentlyQueriedSeriesBlocksConfiguration `yaml:"maxRecentlyQueriedSeriesBlocks"`
+	MaxRecentlyQueriedSeriesBlocks *MaxRecentQueryResourceLimitConfiguration `yaml:"maxRecentlyQueriedSeriesBlocks"`
+
+	// MaxRecentlyQueriedSeriesBytesRead sets the upper limit on time series bytes
+	// read from disk within a given lookback period. Queries which are issued while this
+	// max is surpassed encounter an error.
+	MaxRecentlyQueriedSeriesBytesRead *MaxRecentQueryResourceLimitConfiguration `yaml:"maxRecentlyQueriedSeriesBytesRead"`
```

Comment:
Should we name this "DiskBytesRead"? Just to qualify it without needing to read the docs to work out what specific bytes it's limiting? Also I assume we may add other FooBytesRead/BarBytesRead limits possibly which would require specialization.

Reply:
Makes sense to future-proof the name